.npmrc

```diff
@@ -1,3 +1,2 @@
 registry=https://nexus.ci.diabeloop.eu/repository/npm-diabeloop/
 email=jenkins@diabeloop.fr
-_auth=${nexus_token}
```

Comment:
Not sure, but removing this may block the publication on our nexus repo.
You may replace by:
```
//nexus.ci.diabeloop.eu/repository/npm-diabeloop/:_auth=${NEXUS_TOKEN}
```